model/src/main/java/org/jboss/pnc/model/BuildRecord.java

```diff
@@ -888,6 +896,10 @@ public BuildRecord build() {
             }
             buildRecord.setTemporaryBuild(temporaryBuild);
 
+            if(!temporaryBuild) {
```

Comment:
And another space nissing after "if" :-)

Reply:
We should define the code style more clearly and use some autoformatter. It's written the same way several lines above.